### PR TITLE
Docs bart training ref

### PR DESCRIPTION
**Pull request by lvwerra**
# What does this PR do?
Following the discussion on #7828 with @sshleifer and others I created a minimal example on the forum and added a reference to the Bart documentation. Let me know if this works for you.

### CODE REVIEW

File: docs/source/model_doc/bart.rst [hunk 1]
```diff
@@ -23,7 +23,9 @@ According to the abstract,
   state-of-the-art results on a range of abstractive dialogue, question answering, and summarization tasks, with gains
   of up to 6 ROUGE.
 
-The Authors' code can be found `here <https://github.com/pytorch/fairseq/tree/master/examples/bart>`__.
+The Authors' code can be found `here <https://github.com/pytorch/fairseq/tree/master/examples/bart>`__. An example
+of how to train Bart for conditional generation such as summarisation can be found in `this 
+<https://discuss.huggingface.co/t/train-bart-for-conditional-generation-e-g-summarization/1904>`__ forum discussion.
```
Comment (sgugger) [inline docs/source/model_doc/bart.rst:28]:
```suggestion
of how to train Bart for conditional generation such as summarisation can be found in this `forum discussion
<https://discuss.huggingface.co/t/train-bart-for-conditional-generation-e-g-summarization/1904>`__.
```

**Review by sgugger — APPROVED** [review]
Thanks for the link!

File: docs/source/model_doc/bart.rst (hunk 1 above)
Comment (sshleifer) [reply docs/source/model_doc/bart.rst]: Fixed this on a branch, added other links, and pushed.

File: examples/seq2seq/README.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ## Sequence to Sequence Training and Evaluation
 
 This directory contains examples for finetuning and evaluating transformers on summarization and translation tasks.
-Please tag @sshleifer with any issues/unexpected behaviors, or send a PR!
+Please tag @patil-suraj with any issues/unexpected behaviors, or send a PR!
```
Comment (sshleifer) [inline examples/seq2seq/README.md:4]: cc @patil-suraj

**Comment by sshleifer** [comment]
Thanks for the contribution!
cc @LysandreJik

**Pull request merged**